test/e2e/app-dir/no-react/no-react.test.ts

```diff
@@ -0,0 +1,13 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('no-react', () => {
+  const { next } = nextTestSetup({
+    files: __dirname,
+    installCommand: 'pnpm remove react react-dom',
+  })
+
+  it('should work using cheerio without react, react-dom', async () => {
+    const $ = await next.render$('/')
+    expect($('p').text()).toBe('hello world')
+  })
```

Comment:
How does this test work? Shouldn't it just do a regular `pnpm dev` and then check no `react` or `@types/react` is in dependencies or devDependencies?

Reply:
Since we always install react and react-dom, this manually removes after install.

https://github.com/vercel/next.js/blob/e101bd8de46a917dafcd66213ba545ca964c16d2/test/lib/next-modes/base.ts#L167-L176

![Screenshot 2024-08-17 at 12 15 04 PM](https://github.com/user-attachments/assets/6649a156-3a45-416e-b43c-389d3297a138)

Reply:
But where in the assertion  do I see that?